Tasks/ANT/anttask.ts

```diff
@@ -1,15 +1,18 @@
 /// <reference path="../../definitions/vsts-task-lib.d.ts" />
 
 import tl = require('vsts-task-lib/task');
+import path = require('path');
```

Comment:
We have used path.join always so file path seperator doesnt come into picture here. Will change the task input help to always give linux style seperator.